styleguide/source/_patterns/02-organisms/03-sections/07-simple-step-form/simple-step-form.twig

```diff
@@ -0,0 +1,32 @@
+{#
+ # @file simple-step-form.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<form class="simple-step-form" method="get" action="" novalidate="novalidate">
+  <fieldset>
+    <p class="h2">To reach us by email, please fill out the following:</p>
+    {% include 'molecules-input-text' %}
+    {% include 'molecules-input-telephone' %}
+    {% include 'molecules-input-select' %}
+    {% include 'molecules-input-email' %}
+    {% include 'molecules-input-radio' %}
+    {% include 'molecules-input-month' %}
```

Comment:
Totally agree. This is what's being used on the corp site so I'm not sure if there is any deeper meaning for using the `month` input with that label. @froboy @chelsiejohnston do either of you have insight on this?